python/whylogs/api/writer/s3.py

```diff
@@ -63,15 +63,20 @@ def __init__(
         self.bucket_name = bucket_name or None
```

Comment:
These will set `bucket_name` and `object_name` to `None` if they're passed the empty string. Not sure if that's the intended behavior?